### PR TITLE
creates `/covid` page 

### LINKED ISSUE #267: add COVID safety plan / explanation to website
short & sweet - need to get [this doc](https://docs.google.com/document/d/1zWBY73g0LkqM6qSa1eIhJ5bvt_sGaaKfQqqmqlOSo-U) (or a PDF version) somewhere on the website, with an explanation of how it'll work

### PR DESCRIPTION
**Pull request by mattxwang**
This is a quick-and-dirty PR on behalf of @evanzhong / ops!

This PR does three things:

1. creates a page at `/covid` that links to our Google Doc, a PDF, and embeds an iframe
2. hosts a PDF of the COVID Guidelines 
3. refactors the `TechGala/ViewBanner.js` component to `ContentBanner.js`, and allows props to be passed in for the left side, right side button text, and the link

Closes #267.

### CODE REVIEW

**Comment by mattxwang** [comment]
Netlify preview: https://deploy-preview-269--jovial-pasteur-581b4a.netlify.app/covid

Re: the review, definitely want a content copy check!

File: components/ContentBanner.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
 import Link from 'next/link';
 import React from 'react';
 
-function TGBanner() {
+function ContentBanner({ title, ctaText, ctaLink }) {
```
Comment (evanzhong) [inline components/ContentBanner.js:4]: Clever refactor here

File: pages/covid.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+							alt: 'The ACM at UCLA logo',
+						},
+					],
+					site_name: 'ACM at UCLA',
+				}}
+			/>
+      <div className='content-container-tight'>
+        <h1>ACM at UCLA COVID Policies Fall 2021</h1>
+        <p>
+          Our plan is approved by <a href="https://sole.ucla.edu/" target="_blank" rel="noopener noreferrer">UCLA SOLE</a> and is a superset of <a href="https://covid-19.ucla.edu/ucla-return-to-campus/" target="_blank" rel="noopener noreferrer">UCLA&apos;s return to campus</a> and <a href="http://ph.lacounty.gov/acd/ncorona2019/EducationToolkit/IHE/" target="_blank" rel="noopener noreferrer">LA County Guidelines</a>.
```
Comment (evanzhong) [inline pages/covid.js:29]: Interesting copy re: `superset`, took me a second to really wrap my mind around what you meant by this.

Not sure how to make this more clear though

File: pages/index.js [hunk 3]
```diff
@@ -31,7 +31,11 @@ function Home () {
 				}}
 			/>
 			<div className="home-page text-center">
-				<TGBanner />
+				<ContentBanner
+					title="See how we're taking a safe return-to-campus seriously."
```
Comment (evanzhong) [inline pages/index.js:35]: Thoughts on:
```suggestion
					title="See how we're committing to a safe return-to-campus"
```

**Review by evanzhong — APPROVED** [review]
Approved contingent on some discussion based on comments

A couple of non-blocking nice to haves
- Beveled edges look dated (low priority but should be easy fix) image
- PDF link buttons didn't port extremely well image

Couple (still) non-blocking but larger issues
- Double scroll issue where iframe doesn't comfortably fit in viewport (i.e. as a user do I scroll within the iframe or in the outer page to view the full content) image
- Not enough padding at bottom of screen image

**Comment by mattxwang** [comment]
Resolved 

> Beveled edges look dated (low priority but should be easy fix)

> Not enough padding at bottom of screen

On 

> PDF link buttons didn't port extremely well

Unfortunately, this is on Google - they should be embedding better CSS for the `iframe`. Don't think it's necessarily worth to go in to their CSS, since it's very implementation-dependent.

> Double scroll issue where iframe doesn't comfortably fit in viewport (i.e. as a user do I scroll within the iframe or in the outer page to view the full content)

Yeah, I'm not really sure how to resolve this either, since "zooming out" of the `iframe` is a bit tricky. I could have a more flexible width, but then it aligns poorly with the rest of our content. 

For now, I'll merge; let's tackle any followup in another PR?

**Comment by evanzhong** [comment]
Sounds good to me, I think the biggest issue now is just the iframe.

**Comment by mattxwang** [comment]
> Sounds good to me, I think the biggest issue now is just the iframe.

Yeah, I accidentally messed up and merged this while my push was in-flight; I resolved it on `main`, do you want to take a look?

**Comment by evanzhong** [comment]
If you mean https://github.com/uclaacm/website/commit/9b9b25f442bd31b3d57949176785b41714bd00b0 it lgtm

**Pull request merged**